test/image/mocks/hovermode_xunified.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
You may drop this mock and baseline, if only needed for jasmine test.

Reply:
Done in e82e010da369a347d8d13db18290b34ca710db79